src/styles/blocks/card-stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
```

Comment:
There is an extra space before `:nth-child(-n + 3)`, which may cause the selector to not work as intended. Remove the space to ensure the correct elements are targeted.